tableau_utilities/tableau_server/tableau_server.py

```diff
@@ -739,8 +741,8 @@ def embed_datasource_credentials(self, datasource_id, credentials, connection_ty
 
         for c in self.get_datasource_connections(datasource_id):
             if c.type.lower() == connection_type.lower():
-                c.userName = credentials['username']
+                c.user_name = credentials['username']
```

Comment:
@JustinGrilli  this was the issue with embed password. The old camel case was not changing the values in the dictionary.

If this means something is wrong somewhere else I'll leave it to you to fix.  If this works then it's working now.